frontend/src/components/profile/sidebar/index.jsx

```diff
@@ -0,0 +1,27 @@
+import './sidebar.css';
+
+const Sidebar = () => {
+    return (
+        <div className="sidebar">
+            <ul>
+                <li>
+                    <a href="/my-listing">My Listing</a>
+                </li>
+                <li>
+                    <a href="/list-item">List an Item</a>
+                </li>
+                <li>
+                    <a href="/conversations">Conversations</a>
+                </li>
+                <li>
+                    <a href="/verify-email">Verify Email</a>
+                </li>
+                <li>
+                    <a href="/update-profile">Update Profile</a>
+                </li>
```

Comment:
Anchor tags cause pages to reload when they are clicked. This increases loading time and may lead to data loss from state managers Can we use `Link` from ` react-router-dom` instead? We used Link to route the elements on the navigation bar.